ReactWindows/ReactNative.Shared/UIManager/AccessibleCanvas.cs

```diff
@@ -0,0 +1,30 @@
+#if WINDOWS_UWP
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Automation.Peers;
+#else
+using System.Windows.Controls;
+using System.Windows.Automation.Peers;
+#endif
+
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Canvas with ReactXP/ReactNative accessibility properties support. 
+    /// </summary>
+    public class AccessibleCanvas : Canvas, IAccessible
+    {
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new AccessibleAutomationPeer<AccessibleCanvas>(this);
+        }
+
+        // TODO: implement runtime change raising event to screen reader #1228350
```

Comment:
>1228350 [](start = 74, length = 7)

I like tracking TODOs with issue numbers but we shouldn't include Skype-specific issues numbers in this code since it's open source. Instead, consider creating an issue on the React Native Windows GitHub repo and referencing it here. You can also reference the GitHub issue in the Skype task.